CS328-Notes/notebooks/2022_01_13_chernoff_bound.md

```diff
@@ -0,0 +1,57 @@
+# Lecture 5
+
+### Chernoff Bound
+
+Let $X_{i}$ be the random variable such that probability of $X_{i}=1$ is $p$ and $X_{i}=0$ with probability $(1-p)$ and all $X_{i}$s are independent of each other.
+Let
+
+$$S = \Sigma_{i=1}^{n} X_{i} $$
+
+Then expectation of $S$, $E[S]= \Sigma_{i=1}^{n} E[X_{i}] = np =\mu $
+<!-- <b>Lemma 1. </b> If $$ S=\Sigma_{i=1}^{n} X_{i} $$, where $ X_1, X_2, ..., X_n$ are independent random variables, then.  -->
+
+Note
+
+$$E[e^{tX_{i}}] = pe^t + (1-p) = 1 + p(e^t-1) \leq e^{ p(e^t-1)}$$
+
+for any $y$, $1+y \leq e^y$, here $y=1 + p_{i}(e^{t}-1)$
+
+<b> 1. Upper tail </b>
+
+$$ P(S > (1+\delta)np) \leq \Big(\frac{e^\delta}{(1+\delta)^{(1+\delta)}}\Big)^{np}$$
+
+```{math}
+:label: property_4
+P(S \geq (1+\delta)np) \leq  e^{-\frac{\delta^2}{2+\delta}np}
+```
+
+<b>2. Lower tail </b>
+```{math}
+:label: property_5
+P(S\leq(1-\gamma)np) \leq e^{-\gamma^2 \frac{ np}{2}}
+```
+<b>Proof : </b>
```

Comment:
We can use this `Sphinx-proof` feature. https://sphinx-proof.readthedocs.io/en/latest/syntax.html#proofs

Wrap the proof in ````{prf:proof} .... ````

Reply:
Check other features at: https://sphinx-proof.readthedocs.io/en/latest/syntax.html

Reply:
Revise once before submission 👍

Reply:
I have also added few comments in Grp 1 PR, see if some of them apply here as well.